cli/src/pcluster/api/pcluster_api.py

```diff
@@ -339,14 +338,11 @@ def delete_image(image_id: str, region: str, force: bool = False):
                 os.environ["AWS_DEFAULT_REGION"] = region
             # retrieve imagebuilder config and generate model
             imagebuilder = ImageBuilder(image_id=image_id)
+            image = PclusterApi._get_underlying_image_or_stack(imagebuilder, image_id)[0]
```

Comment:
isn't better 
`image, _ = PclusterApi._get_underlying_image_or_stack(imagebuilder, image_id)`
? I think it's more readable

Reply:
done